### PR TITLE
changed sessions views (for login and signup), routes (not working)

### PR DESCRIPTION
**Pull request by amarrap**

### CODE REVIEW

File: app/views/sessions/login.html.erb [hunk 1]
```diff
@@ -1,2 +1,37 @@
+<form>
+  <!-- Username input -->
+  <div class="form-outline mb-4">
```
Comment (kvangreen) [inline app/views/sessions/login.html.erb:3]: Consider changing some of these divs to semantic tags

File: app/views/sessions/login.html.erb [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+      <!-- Checkbox -->
+      <div class="form-check">
+        <input class="form-check-input" type="checkbox" value="" id="form2Example31" checked />
+        <label class="form-check-label" for="form2Example31"> Remember me </label>
+      </div>
+    </div>
+
+    <div class="col">
+      <!-- Simple link -->
+      <a href="#!">Forgot password?</a>
```
Comment (kvangreen) [inline app/views/sessions/login.html.erb:26]: Assuming this does not have functionality yet

File: app/views/sessions/signup.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <h1>New</h1>
 
-<h1>Login</h1>
+<h1>Sign Up</h1>
    <%= form_tag '/login' do %>
    <%= label_tag :username%>
    <%= text_field_tag :username %>
```
Comment (kvangreen) [inline app/views/sessions/signup.html.erb:6]: Does this need a form tag? Also, were we planning on adding more fields?

File: app/views/sessions/signup.html.erb [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <h1>New</h1>
 
-<h1>Login</h1>
+<h1>Sign Up</h1>
```
Comment (kvangreen) [inline app/views/sessions/signup.html.erb:3]: Either this should become h2, or we should get rid of line 1 OR line 3

File: app/views/sessions/welcome.html.erb [hunk 5]
```diff
@@ -19,7 +19,7 @@
 
 <%= button_to "Login", '/login', method: :get%>
```
Comment (kvangreen) [inline app/views/sessions/welcome.html.erb:20]: What's the difference between link_to and button_to?

File: app/views/stations/index.html.erb [hunk 6]
```diff
@@ -1,16 +1,59 @@
 <div class="page-section flexbox vertical stretch">
```
Comment (kvangreen) [inline app/views/stations/index.html.erb:1]: What functionality is this page serving with welcome.html.erb?

File: app/views/stations/index.html.erb [hunk 7]
```diff
@@ -1,16 +1,59 @@
...
+
+            // When ready to go live, remove the alert and show a success message within this page. For example:
+            // var element = document.getElementById('paypal-button-container');
+            // element.innerHTML = '';
+            element.innerHTML = '<h3>Thank you for your payment!</h3>';
+            // Or go to another URL:  actions.redirect('thank_you.html');
+          });
+        },
+
+        style: {
```
Comment (kvangreen) [inline app/views/stations/index.html.erb:44]: This should go in a style sheet

File: app/views/users/create.html.erb [hunk 8]
```diff
@@ -1 +1 @@
-<h1>Users#create</h1>
+<h1>Users</h1>
```
Comment (kvangreen) [inline app/views/users/create.html.erb:1]: Is this redundant with /sessions/signup.html.erb (or /login.html.erb)?

File: app/views/users/new.html.erb [hunk 9]
```diff
@@ -1,5 +1,3 @@
-<h1>Users#new</h1>
-
 <h1>Sign Up</h1>
```
Comment (kvangreen) [inline app/views/users/new.html.erb:1]: Is this redundant with /sessions/signup.html.erb?

File: app/views/sessions/pricing.html.erb [hunk 10]
```diff
@@ -1 +1,2 @@
 <h1>Pricing</h1>
+<h3>give us money</h3>
```
Comment (epartakki) [inline app/views/sessions/pricing.html.erb]: smooth!

File: app/views/sessions/signup.html.erb (hunk 4 above)
Comment (epartakki) [inline app/views/sessions/signup.html.erb:3]: oops, you're right, my bad!

**Review by luciebrock — APPROVED** [review]
looks good... I guess

File: payment.html [hunk 11]
```diff
@@ -0,0 +1,45 @@
+<div id="smart-button-container">
```
Comment (kvangreen) [inline payment.html:1]: Where is this file?

**Review by kvangreen — APPROVED** [review]
Good

**Review by epartakki — APPROVED** [review]
Good start working on this Aline!

**Review by sunniraleigh — APPROVED** [review]
i approve

**Pull request merged**